auto_submit/lib/validations/autosubmit_label_existence.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:auto_submit/model/auto_submit_query_result.dart';
+import 'package:auto_submit/validations/validation.dart';
+import 'package:github/github.dart' as github;
+
+import '../service/config.dart';
+
+/// Validates that the list of checks for the PR is not empty.
+class AutosubmitLabelExistence extends Validation {
+  AutosubmitLabelExistence({
+    required Config config,
+  }) : super(config: config);
+
+  @override
+
+  /// The `autosubmit` label must exist when landing PRs.
+  Future<ValidationResult> validate(QueryResult result, github.PullRequest messagePullRequest) async {
+    // List of labels associated with the pull request.
+    final List<String> labelNames = (messagePullRequest.labels as List<github.IssueLabel>)
```

Comment:
Do we need to get the labels directly using the github client? as it can still land the PR if the message was queued before the label was removed.

Reply:
Switched the logic to `shouldProcess`, which handles this naturally.